dabplus-on-air-processing.spec

```diff
@@ -11,7 +11,7 @@ Name:     dabplus-on-air-processing
 BuildRequires: make
 BuildRequires: systemd
 
-Requires: liquidsoap
+Requires: liquidsoap = ${_ls_version}
```

Comment:
> The version needs to be exact since liquidsoap always expects its libs to be in the liqdir folder which is version specific.

I guess, it makes sense then - at least if you continue to use `%include <file>` directives which are relative to the library directory of liquidsoap instead of the `%include "file"` directive which could be in a independent non-versioned directory.